servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/DefaultNettyConnection.java

```diff
@@ -626,6 +646,9 @@ private void completeSubscriber() {
             assert subscriber != null;
             SingleSource.Subscriber<? super DefaultNettyConnection<Read, Write>> subscriberCopy = subscriber;
             subscriber = null;
+            if (observer != null) {
+                connection.readWriteObserver(requireNonNull(observer.established(connection)));
```

Comment:
should we be more defensive here and account for exceptions? we are invoking user code that may throw and break async control flow notification to `subscriberCopy` IIUC.